Next____

igor-zhylin opened the pull request.

igor-zhylin committed:
BaseStructureUpdated

Added DLL and create class list

igor-zhylin committed:
First Update

Added :
Add
Insert
Length

igor-zhylin committed:
Added Remove By Id

Olga-Levchenko commented on ObjectList/List.cs:
```diff
@@ -0,0 +1,110 @@
+
+        public object this[int index]
+        {
+            get { return _array[index]; }
```
Here you should check if the index falls within the size

Olga-Levchenko commented on ObjectList/List.cs:
```diff
@@ -0,0 +1,110 @@
+            }
+            else
+            {
+                ResizeErray();
```
Erray -> Array

Olga-Levchenko commented on ObjectList/List.cs:
```diff
@@ -0,0 +1,110 @@
+            }
+        }
+
+        private void CheckObjectZeroValues(object obj)
```
It's something like 'CheckForNull'

Olga-Levchenko commented on ObjectList/List.cs:
```diff
@@ -0,0 +1,110 @@
+
+        public void Remove(int index)
+        {
+            if (index > Length)
```
I suppose it's 'index >= Length'

Olga-Levchenko reviewed (commented):
The code is clean and logical. Good job!

igor-zhylin committed:
Update rev 1.1

Added methods:
1. CheckIndex{return index or throw IndexException}
2. IndexOf
3. Reverse
4. Clear
